### DISCUSSION TITLE
I want to know the exact time of release tag

### DISCUSSION THREAD

**Discussion by KyleBing** [community/community#46210] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I want to know the exact time of the release tag, instead of the count of days after its release date.

![2023-02-03 09 10 55](https://user-images.githubusercontent.com/12215982/216487833-da6072ce-d2f3-4274-b19a-9bc14df37e8c.png)

**Comment by jge162** [comment]
Do you own the repo? you may need to contact the developers or maintainers of the project to ask for more information.

**Comment by yusancky** [comment, marked as answer]
### 中文 / Chinese

从我这里看，时间并不是直接显示为 `X days / weeks ago`，而是先显示具体时间，然后再在约 0.4 秒后才显示为计时格式。可能这是由某些脚本来处理的，可能可以考虑禁用脚本？

我正在使用 iPad，但我记得电脑上的一些场景下，可以 **通过鼠标悬停显示具体时间**，不知道这里怎么样。

----
### English / 英文

> **Note**
> If your computer does not display the text above, it may be because the CJK font is missing. The English at the bottom has the same content as the Chinese. 

From what I can see here, the time is not displayed directly as `X days / weeks ago`, but the exact time is displayed first, and then only after about 0.4 seconds in chronological format. Maybe this is handled by some script, maybe consider disabling the script? 

I'm using an iPad, but I remember some scenarios on PC where you can **display the exact time by hovering over it**, not sure how it works here.

**Reply by KyleBing** [reply]
没有使用任何脚本， github 对于近几天的时间就显示为 x days， 远了就显示时间了。

试了下，悬停确实能显示具体时间，棒！

![image](https://user-images.githubusercontent.com/12215982/216492275-ba9906e5-9526-4cf2-b2f5-af5e1591ffac.png)
